www-dev/index.html

```diff
@@ -14,14 +14,24 @@
         var frame = '<iframe id="chat" src="chat.html{{params}}" style="height: calc(100% - 23px); width: calc(100% - 20px); padding: 10px; border:none;"></iframe>';
         
         function auth() {
+            var pass;
+            var chan;
             var user = prompt("Please enter your username, or leave blank to use without logging in");
             if (!user) {
-                document.body.innerHTML += frame.replace('{{params}}', '');
-                return;
+                chan = prompt("If you want to go into a channel other than global (ie, if you're going to be doing testing that could potentially disturb chat), please type it here (you might want to use \"test\" for such a case). Otherwise, just click ok. (In global you can only look at chat, but in other channels (such as \"test\") you will be able to write)");
+                if(chan && chan != "global"){
+                    user = "GuestForTest";
+                    pass = "12345";
```

Comment:
I don't see a need to set a password - if the user doesn't log in, there's no need to validate the user. This does of course mean that either the login request (line 35) would need to be bypassed. Maybe have a function for generating the iframe URL, so both here and in the request you could call something like `insertFrame(username, uid, channel`